### PR TITLE
Enable migration tests to run on GKE clusters.

### PR DESCRIPTION
**Pull request by TeweiLuo**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test

/kind feature
> /kind flake

**What this PR does / why we need it**:
CSIMigrationGCE can be enabled in GKE Test env now. This PR will let us add Prow configs to run migration tests on GKE.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/run-k8s-integration.sh [hunk 1]
```diff
@@ -25,9 +25,12 @@ readonly gke_release_channel=${GKE_RELEASE_CHANNEL:-""}
 readonly teardown_driver=${GCE_PD_TEARDOWN_DRIVER:-true}
 readonly gke_node_version=${GKE_NODE_VERSION:-}
 readonly use_kubetest2=${USE_KUBETEST2:-false}
+readonly migration_test=${MIGRATION_TEST:-false}
 
 export GCE_PD_VERBOSITY=9
 
+readonly GCE_PD_TEST_FOCUS="PersistentVolumes\sGCEPD|[V|v]olume\sexpand|\[sig-storage\]\sIn-tree\sVolumes\s\[Driver:\sgcepd\]|allowedTopologies|Pod\sDisks|PersistentVolumes\sDefault"
```
Comment (mattcary) [inline test/run-k8s-integration.sh:32]: Do you know why we have the restricted focus for migration tests?

Comment (TeweiLuo) [reply test/run-k8s-integration.sh:32]: David wrote this focus a while ago - are there certain in-tree tests you noticed that are not covered here?

Comment (mattcary) [reply test/run-k8s-integration.sh:32]: I have no idea. I just wondered if it was a copy-paste or if we know positively that it covers all the in-tree tests.

Comment (TeweiLuo) [reply test/run-k8s-integration.sh:32]: It's a copy-n-paste. I should probably learn more about the tests.

Comment (TeweiLuo) [reply test/run-k8s-integration.sh:32]: I went through the test cases, especially those in this testgrid https://k8s-testgrid.appspot.com/sig-storage-kubernetes#gce.

**Pull request merged**